playbooks/move_vm_from_on_prem_to_aws/README.md

```diff
@@ -0,0 +1,78 @@
+
+AWS User Account with the following permissions:
+* s3:GetBucketLocation
+* s3:GetObject
+* s3:ListBucket
+* s3:GetBucketLocation
+* s3:GetObject
+* s3:ListBucket
+* s3:PutObject
+* s3:GetBucketAcl
+* ec2:ModifySnapshotAttribute
+* ec2:CopySnapshot
+* ec2:RegisterImage
+* ec2:Describe*
+* ec2:RunInstances
+
+(Optional) To import resources encrypted using an AWS KMS key from AWS Key Management Service, add the following permissions:
+* kms:CreateGrant
+* kms:Decrypt
+* kms:DescribeKey
+* kms:Encrypt
+* kms:GenerateDataKey*
+* kms:ReEncrypt*
+
+## Playbook Variables
+
+### Needed for the cloud.aws_ops.clone_on_prem_vm role
+
+* **on_prem_source_vm_name** (str): (Required) The name of the on-prem VM you want to clone.
+* **on_prem_vm_clone_name** (str): (Optional) The name you want to call the cloned image. If not set, the the **on_prem_vm_clone_name** will be used with a _-clone_ suffix.
+* **uri** (str): (Optional) Libvirt connection uri. Default: "qemu:///system".
+* **overwrite_clone** (bool): (Optional) Weather to overwrite or not an already existing on prem VM clone. Default: true.
+
+### Needed for the cloud.aws_ops.clone_on_prem_vm role
+
+* **aws_access_key** (str): (Required) AWS access key ID for user account with the above permissions
+* **aws_secret_key** (str): (Required) AWS secret access key for user account with the above permissions
+* **aws_region** (str): (Required) AWS region in which to run the EC2 instance
+* **security_token** (str): (Optional) Security token for AWS session authentication
+* **s3_bucket_name** (str): (Required) The name of the S3 bucket name where you want to upload the .raw image. It must exist in the region the instance is created.
+* **import_task_name** (str): (Required) The  name you want to assign to the AWS EC2 import image task.
+* **image_path** (str): (Required) The path where the .raw image is stored.
+* **instance_name** (str): (Required) The name of the EC2 instance you want to create using the imported AMI.
+* **instance_type** (str): (Optional) The EC2 instance type you want to use. Default: "t2.micro".
+* **keypair_name** (str): (Optional) The name of the SSH access key to assign to the EC2 instance. It must exist in the region the instance is created. If not set, your default AWS account keypair will be used.
+* **security_groups** (list): (Optional) A list of security group IDs or names to assiciate to the EC2 instance.
+* **vpc_subnet_id** (str): (Optional) The subnet ID in which to launch the EC2 instance instance (VPC). If none is provided, M(amazon.aws.ec2_instance) will chose the default zone of the default VPC.
+* **instance_volumes** (dict): (Optional) A dictionary of a block device mappings, by default this will always use the AMI root device so the **instance_volumes** option is primarily for adding more storage. A mapping contains the (optional) keys _device_name_, _ebs.volume_type_, _ebs.volume_size_, _ebs.kms_key_id_, _ebs.iops_, and _ebs.delete_on_termination_.
```

Comment:
This would benefit from having the same structure as you use for the `kvm_host` var.